starport/services/chain/build.go

```diff
@@ -89,13 +89,26 @@ func (c *Chain) BuildRelease(ctx context.Context, prefix string, targets ...stri
 		return "", err
 	}
 
-	releasePath = filepath.Join(c.app.Path, releaseDir)
+	binary, err := c.Binary()
+	if err != nil {
+		return "", err
+	}
+
+	mainPath, err := goanalysis.DiscoverMain(c.app.Path)
+	if err != nil {
+		return "", err
+	}
+
+	releasePath = output
+	if output == "" {
+		releasePath = filepath.Join(c.app.Path, releaseDir)
+	}
 
 	// reset the release dir.
 	if err := os.RemoveAll(releasePath); err != nil {
```

Comment:
```suggestion
```

since the --output is configurable now, this line is a bit dangerous. e.g. --output $HOME